helm/ovn-kubernetes-dpf/values.yaml

```diff
@@ -95,3 +95,7 @@ serviceDaemonSet:
   #  type: RollingUpdate
   #  rollingUpdate:
   #    maxUnavailable: 1
+
+# lease namespace should be the same as the where the DPU node lease is created
+# default value is unset, which means the namespace is the same as the release namespace
+leaseNamespace: ""
```

Comment:
you need to update the values.yaml.tmpl (also nit: remove line at the end of the file)

Reply:
nit: might be worth making that `nil` instead of ""